### PR TITLE
Add Missing institutions

### PR DESCRIPTION
**Pull request by verwoerd**

### CODE REVIEW

File: universities/organizations.json [hunk 1]
```diff
@@ -449,6 +776,18 @@
     "formal_name": "University of Luxembourg",
     "country": "LUX"
   },
+  {
+    "id": "14148",
+    "icpc_id": "14148",
+    "name": "UoL",
```
Comment (mpsijm) [inline universities/organizations.json:782]: Duplicate with "University of Liverpool", but this data probably comes from ICPC so then it should be changed elsewhere :stuck_out_tongue:

**Review by mpsijm — APPROVED** [review]

File: universities/organizations.json (hunk 1 above)
Comment (verwoerd) [reply universities/organizations.json:782]: correct, short name is also not required to be unique.

File: universities/organizations.json [hunk 2]
```diff
@@ -361,33 +630,87 @@
...
+    "location": {
+      "longitude": 25.287514,
+      "latitude": 54.682789
+    }
+  },
+  {
+    "id": "7488",
+    "icpc_id": "7488",
+    "name": "NUI Maynooth",
+    "formal_name": "Maynooth University",
```
Comment (mpsijm) [inline universities/organizations.json:683]: Should it also be added to the Readme? 🙂

Comment (verwoerd) [reply universities/organizations.json:683]: Forgot about this. Will run the table generate script in a later commit again.

**Review by mpsijm — APPROVED** [review]

File: universities/organizations.json [hunk 3]
```diff
@@ -861,6 +861,18 @@
       "latitude": 50.87959
     }
   },
+  {
+    "id": "11352",
+    "icpc_id": "11352",
+    "name": "RUB",
+    "formal_name": "Ruhr University",
```
Comment (mpsijm) [inline universities/organizations.json:868]: For the "kick up the totem pole list": It looks like their official name includes "Bochum" :stuck_out_tongue: 

- https://www.ruhr-uni-bochum.de/en
- https://en.wikipedia.org/wiki/Ruhr_University_Bochum

Cool to see a new name! :smile:

**Pull request merged**